audio/faust-devel/Portfile

```diff
@@ -8,6 +8,7 @@ github.setup            grame-cncm faust 96f22a55e86dac30cbf2e38ba115f5723ed6b09
 # When updating faust-devel to a new version, please rebuild faustlive-devel
 # simultaneously by increasing its revision or updating it to a new version.
 version                 2.11-20181109
+revision                1
```

Comment:
No needs for revision increase since this is adding a build dependency. It won't build for someone without it.

Reply:
Fixed